### PR TITLE
Update 'save' and 'load' commands in User Guide

### LINKED ISSUE #126: Update 'save' and 'load' commands in User Guide

### PR DESCRIPTION
**Pull request by ChenLeiyu**
fixes #126

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,9 +3,11 @@
   title: "User Guide"
   pageNav: 3
 ---
-# Gamerbook PRO MAX User Guide
+# GamerBook PRO MAX User Guide
```
Comment (JJtan2002) [inline docs/UserGuide.md:6]: Should be GamerBook Pro Max, without the full capitalisation.

**Review by JJtan2002 — CHANGES_REQUESTED** [review]
Please make the changes as needed according to my comments, thank you Lei Yu.

File: docs/UserGuide.md (hunk 1 above)
Comment (ChenLeiyu) [reply docs/UserGuide.md]: Done, thank you for the review.

**Review by JJtan2002 — APPROVED** [review]
LGTM!

**Pull request merged**